rpc/src/v1/traits/parity.rs

```diff
@@ -218,5 +218,9 @@ build_rpc_trait! {
 		/// Call contract, returning the output data.
 		#[rpc(name = "parity_call")]
 		fn call(&self, Vec<CallRequest>, Trailing<BlockNumber>) -> Result<Vec<Bytes>>;
+
+		/// ecrecover signature
+		#[rpc(name = "parity_verifySignature")]
+		fn verify_signature(&self, bool, Bytes, H256, H256, u64) -> Result<RichBasicAccount>;
```

Comment:
I still don't understand why we need `RichBasicAccount` here, if we don't pass any "rich" data. Just return the `BasicAccount` and get rid of the `to_rich_struct` helper func.